src/js/actions/markActions.js

```diff
@@ -168,6 +170,13 @@ function bindField(id, field, property) {
   };
 }
 
+function removeAxisFromGroup(axisId, groupId) {
```

Comment:
Rather than a separate `REMOVE_AXIS_FROM_GROUP` action, let's have the mark reducer reuse the `DELETE_GUIDE` action from above. This reduces the chance of erroneously calling only `DELETE_GUIDE` without `REMOVE_AXIS_FROM_GROUP`. To be able to do this, we'll want to make sure `DELETE_GUIDE` has all the necessary information for the marks reducer (namely the groupId and guide type (axis/legend)).